region/lowernorfair/west.json

```diff
@@ -418,9 +418,17 @@
                     ]}
                   ]}
                 ]}
+              ]},
+              {"and": [
+                "heatProof",
+                {"or":[
+                  "Charge",
```

Comment:
I promise you, you don't want to kill GT with only Charge even if you're heatproof.